azure-pipelines.yml

```diff
@@ -0,0 +1,154 @@
+stages:
+- stage: build
+  jobs:
+    - job: build_rust
+      variables:
+        build_command: 'cargo build'
+        test_command: 'cargo test'
+      strategy:
+        matrix:
+          mac-stable-combadge:
+            imageName: 'macos-10.13'
+            rustup_toolchain: stable
+            working_directory: combadge
+            binary_file: target/release/combadge
+            artifact_name: combadge_mac
+          linux-stable-combadge:
+            imageName: 'ubuntu-16.04'
+            rustup_toolchain: stable
+            working_directory: combadge
+            binary_file: target/x86_64-unknown-linux-musl/release/combadge
+            artifact_name: combadge_linux
+            build_command: 'cross build --target x86_64-unknown-linux-musl'
+            test_command: 'cross test --target x86_64-unknown-linux-musl'
+          win-stable-combadge:
+            imageName: 'vs2017-win2016'
+            rustup_toolchain: stable
+            working_directory: combadge
+            binary_file: target/release/combadge.exe
+            artifact_name: combadge_windows
+          linux-stable-transporter:
+            imageName: 'ubuntu-16.04'
+            rustup_toolchain: stable
+            working_directory: transporter
+            binary_file: target/release/transporter
+            artifact_name: transporter
+
+      pool:
+        vmImage: $(imageName)
+
+      steps:
+        - script: |
+            curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain $RUSTUP_TOOLCHAIN
+            echo "##vso[task.setvariable variable=PATH;]$PATH:$HOME/.cargo/bin"
+          displayName: Install rust
+          condition: ne( variables['Agent.OS'], 'Windows_NT' )
+        - script: |
+            curl -sSf -o rustup-init.exe https://win.rustup.rs
+            rustup-init.exe -y --default-toolchain %RUSTUP_TOOLCHAIN%
+            echo "##vso[task.setvariable variable=PATH;]%PATH%;%USERPROFILE%\.cargo\bin"
+          displayName: Windows install rust
+          condition: eq( variables['Agent.OS'], 'Windows_NT' )
+          # Until https://github.com/rust-embedded/cross/pull/169 is merged
+        - script: |
```

Comment:
All of this should be replaced with a simple `cargo install cross` once a new version is released. Master already includes the fix.